src/controller/ApplicationController.tsx

```diff
@@ -308,6 +309,13 @@ function StartControllerScope<
       }
     }
 
+    let ControlledViewComponent = ControlledComponent;
+    // @ts-expect-error These are react-easy-state specific props to tell if the component is already memoized
+    if (ControlledViewComponent.type?.name !== 'ReactiveComp') {
```

Comment:
I'm not sure if this additional complexity is worth this change, it could potentially break with different React versions? It seems to be relying on internals.

Reply:
As an alternative, we could add a property to the compoent within ApplicationView, something like 

```
function ApplicationView<T extends ComponentType>(component: T): T {
  component.__ahaMVCView = true;
  return view(component);
}
```
And check that instead